sailthru/requirements/base.txt

```diff
@@ -1,2 +1,3 @@
+nose==1.3.7
```

Comment:
@mikedikan it's the latest version of [nose](https://pypi.python.org/pypi/nose/) available. I would have suggested https://pypi.python.org/pypi/pytest, but that's besides the point.